Failed: FlowThrottleSpec

https://jenkins.akka.io:8498/job/akka-nightly/4907/testReport/junit/akka.stream.scaladsl/FlowThrottleSpec/Throttle_for_various_cost_elements_must_burst_according_to_its_maximum_if_enough_time_passed/

```
sbt.ForkMain$ForkError: java.lang.AssertionError: assertion failed: received unexpected message RealMessage(OnNext(6),Actor[akka://FlowThrottleSpec/deadLetters]) after 49 millis
	at scala.Predef$.assert(Predef.scala:170)
	at akka.testkit.TestKitBase$class.expectNoMsg_internal(TestKit.scala:684)
	at akka.testkit.TestKitBase$class.expectNoMsg(TestKit.scala:652)
	at akka.testkit.TestKit.expectNoMsg(TestKit.scala:882)
	at akka.stream.testkit.TestSubscriber$ManualProbe.expectNoMsg(StreamTestKit.scala:588)
	at akka.stream.scaladsl.FlowThrottleSpec$$anonfun$2$$anonfun$apply$mcV$sp$18$$anonfun$apply$32.apply(FlowThrottleSpec.scala:250)
	at akka.stream.scaladsl.FlowThrottleSpec$$anonfun$2$$anonfun$apply$mcV$sp$18$$anonfun$apply$32.apply(FlowThrottleSpec.scala:238)
```

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowThrottleSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowThrottleSpec.scala
@@ -124,10 +124,6 @@ class FlowThrottleSpec extends StreamSpec {
       (1 to 5) foreach upstream.sendNext
       downstream.receiveWithin(300.millis, 5) should be(1 to 5)
 
-      downstream.request(1)
-      upstream.sendNext(6)
-      downstream.expectNoMsg(100.millis)
-      downstream.expectNext(6)
       downstream.request(5)
       downstream.expectNoMsg(1200.millis)
       for (i ← 7 to 11) upstream.sendNext(i)

